patterns/template-portfolio-home.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Title: Portfolio Template
+ * Slug: twentytwentyfour/template-portfolio-home
```

Comment:
This might be bad English but should it be template-home-portfolio instead? leaving the "portfolio" bit to last as a descriptor of what kind of home template we are referring to

Reply:
Yea I like that. Will change.